internal/payloads/error_payload.go

```diff
@@ -43,6 +49,8 @@ func NewInvalidRequestError(ctx context.Context, message string, err error) *Res
 		Message:        message,
 		TraceId:        ctxval.TraceId(ctx),
 		Error:          err,
+		Version:        version.APIPathVersion,
```

Comment:
I missed this, should have been `version.BuildCommit` because the APIPathVersion is expected to be "v1" for a long time.